### PR TITLE
Remove TestRetort

### PR DESCRIPTION
**Pull request by lubaskinc0de**
Removing TestRetort for lack of need, and because it may not obviously work and may cause errors. (suggested by @zhPavel)

### CODE REVIEW

File: tests/unit/morphing/generic_provider/test_literal_provider.py [hunk 1]
```diff
@@ -4,22 +4,15 @@
...
 @pytest.fixture()
 def retort():
-    return TestRetort(
-        recipe=[
-            LiteralProvider(),
-            EnumExactValueProvider(),
-            UnionProvider(),
-        ],
-    )
+    return Retort()
```
Comment (zhPavel) [inline tests/unit/morphing/generic_provider/test_literal_provider.py:15]: This fixture can be eliminated

File: tests/unit/morphing/generic_provider/test_union_provider.py [hunk 2]
```diff
@@ -17,36 +16,9 @@ class Book:
...
-        recipe=[
-            UnionProvider(),
-            make_loader(str),
-            make_loader(int),
-            make_dumper(str),
-            make_dumper(int),
-            make_dumper(type(None)),
-        ],
-    )
+    return Retort()
```
Comment (zhPavel) [inline tests/unit/morphing/generic_provider/test_union_provider.py:21]: This fixture can be eliminated

File: tests/unit/morphing/test_iterable_provider.py [hunk 3]
```diff
@@ -46,23 +46,23 @@ def retort():
     )
```
Comment (zhPavel) [inline tests/unit/morphing/test_iterable_provider.py:46]: Other tests should use Retort, only test_mapping_providing requires own set of providers

File: tests/unit/morphing/test_iterable_provider.py [hunk 4]
```diff
@@ -46,23 +46,23 @@ def retort():
...
+@pytest.mark.parametrize(
+    "mapping_type",
+    [
+        (dict,),
+        (Dict,),
+        (Mapping,),
+        (collections.Counter,),
+    ],
+)
+def test_mapping_providing(retort, strict_coercion, debug_trail, mapping_type):
```
Comment (zhPavel) [inline tests/unit/morphing/test_iterable_provider.py:58]: Is mapping_type will contain tuples of one element?

**Review by zhPavel — CHANGES_REQUESTED** [review]

File: tests/unit/morphing/test_iterable_provider.py (hunk 3 above)
Comment (lubaskinc0de) [reply tests/unit/morphing/test_iterable_provider.py:46]: +

File: tests/unit/morphing/generic_provider/test_literal_provider.py (hunk 1 above)
Comment (lubaskinc0de) [reply tests/unit/morphing/generic_provider/test_literal_provider.py]: +

File: tests/unit/morphing/test_iterable_provider.py [hunk 5]
```diff
@@ -37,32 +36,26 @@ def string_dumper(data):
...
-            STR_LOADER_PROVIDER,
             dumper(str, string_dumper),
         ],
     )
 
 
-def test_mapping_providing(retort, strict_coercion, debug_trail):
-    retort = retort.replace(
+@pytest.mark.parametrize("mapping_type", [dict, Dict, Mapping, collections.Counter])
+def test_mapping_providing(retort, strict_coercion, debug_trail, mapping_type):
```
Comment (zhPavel) [inline tests/unit/morphing/test_iterable_provider.py:47]: ``retort`` fixture does not used here

**Review by zhPavel — CHANGES_REQUESTED** [review]

**Review by zhPavel — APPROVED** [review]

**Pull request merged**